setup.py

```diff
@@ -30,12 +30,14 @@
         'License :: OSI Approved :: MIT License',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6'
+        'Programming Language :: Python :: 3.7'
+        'Programming Language :: Python :: 3.8'
         # yapf: enable
     ],
     license='License :: OSI Approved :: MIT License',
     keywords='design-by-contract precondition postcondition validation',
     packages=find_packages(exclude=['tests']),
-    install_requires=['asttokens>=1,<2'],
+    install_requires=['asttokens>=1,<3'],
```

Comment:
Should probably be `>=2`, since version 1 has several bugs in it, especially for 3.8.

The reason for the major version change is that tuples now include surrounding parentheses: https://github.com/gristlabs/asttokens/commit/93f1b2d816bf075aa02b6b9ef2db2d6e1a6221cb

Is that a problem for you?

Reply:
Hi @alexmojaki,
No, actually that's not a problem, I'll change the asttokens version tonight.